charter.md

```diff
@@ -53,15 +53,227 @@ The following responsibilities belong to the Steering Committe, but may be deleg
+self-elected reason, they must notify the committee in writing.
+
+### Dissolution
+
+If committee members believe that the committee is no longer able to act in
+accordance with the above [mission](#mission), the committee may dissolve by a
+few methods:
+
+* Dissolution by vote
+* Dissolution by loss of committee
+
+See [elections](#elections) for special criteria in the instance of a general
+election triggered by dissolution.
+
+#### Continuity
+
+If the committee is dissolved completely by any mechanism, selected members
+(selection described below) agree to be "continuity advisors" and attend
+meetings of the subsequently elected incoming committee in a non-voting
+capacity for 6 meetings or until the incoming committee dismisses them,
+whichever is first.
+
+##### Selection
+
+At least two (2) and not more than three (3) members shall be selected for
+this duty.
+
+* One shall be the longest-serving outgoing committee member. If multiple, the
+  incoming committee may select between them.
+* The second shall be selected by the incoming committee at their discretion.
+* The incoming committee, at their discretion, may select a third.
+
+#### Dissolution by vote
+
+Dissolution requires an affirmative vote of a **_two-thirds supermajority of
+the [fixed membership of the committee](#composition)_**, and must be unanimous
+when the committee has four (4) or fewer members.
+
+Example:
+
+* 7 (members) / 3 = 2.333...
+* 2.333... * 2 = 4.666...
+* Round up to the nearest whole number (5)
+* An affirmative vote of 5 members would be required to dissolve the committee
+
+The committee must specify a date of dissolution.
+If the committee is dissolved, all seats are vacated on the date specified.
+
+#### Dissolution by loss of committee
+
+If at any time a **_majority of the
+[fixed membership of the committee](#composition)_**'s seats are vacant, the entire committee will be dissolved.
+
+Example:
+
+* 7 (seats) / 2 = 3.5
+* Round up to the nearest whole number (4)
+* If 4 committee seats are vacant, the entire committee will dissolve
+
+The number of vacant seats will be evaluated prior to any replacement of
+previously-seated members.
+
+### Removal
+
+#### No confidence
+
+A Steering Committee member may be removed by a three-fourths vote of no
+confidence from other Steering Committee members.
+
+The call for a vote of no confidence will happen in a public Steering Committee meeting and a
+GitHub issue in kubernetes/steering by a current member and must be seconded by
+another current member.
+
+Once a vote of no confidence has been called, the committee will notify the
+community through the following channels:
+
+* the [community mailing list][dev-list]
+* the [Steering Committee public mailing list][steering-public-list]
+
+This notification should include a link to the aforementioned GitHub issue.
+
+There will be a period of one week for members of the community to reach
```

Comment:
```suggestion
There will be a period of four weeks for members of the community to reach
```

One week looks short. The rule-making process is four weeks without unanimous agreement to vote early, the meeting cadence is monthly, this stands out as an unusually short time frame. Four weeks lines up well with both policies.

Reply:
Updated to two weeks to reflect the conversation about comment periods from a previous review:

From @lavalamp in https://github.com/kubernetes/steering/pull/248#discussion_r911469884:

> I was going to say a week seems OK, but it'd really suck to have a board pull this on a member who is on vacation or something and can't respond. Personally I'd probably go longer, maybe 2 weeks.

From @dims in https://github.com/kubernetes/steering/pull/248#discussion_r911473383:

> @lavalamp feels reasonable to me. I'd stick to private feedback mechanism, just email steering private so everyone on steering sees it including the person in question.
> 
> yes you are right on holidays/vacations ... so 2 weeks.